recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ requirements:
     - hatch-vcs >=0.2.0
     # for `conda init` in build/script above
     - boltons >=23.0.0
+    - frozendict >=2.4.2
```

Comment:
Is this required for the build? It's already listed as a runtime dep.

Reply:
Yes it is!